src/app/components/LiveHeaderMedia/index.styles.tsx

```diff
@@ -30,12 +30,18 @@ export default {
       position: 'relative',
       padding: 0,
     }),
-  openButton: () =>
+  openButton: ({ mq }: Theme) =>
     css({
       cursor: 'pointer',
       backgroundColor: 'unset',
       border: 'unset',
       textAlign: 'start',
+      [mq.GROUP_2_MAX_WIDTH && mq.GROUP_3_MAX_WIDTH && mq.GROUP_4_MAX_WIDTH]: {
+        marginTop: '0.75rem',
+      },
+      [mq.GROUP_5_MIN_WIDTH]: {
+        marginTop: '0.75rem',
+      },
```

Comment:
I believe we can use `GROUP_3_MIN_WIDTH`, which will cater for all of these scenarios, so that we don't need to declare it multiple times

```suggestion
      [mq.GROUP_3_MIN_WIDTH]: {
        marginTop: '0.75rem',
      }
```